README.md

```diff
@@ -26,38 +26,144 @@ guarantee these changes will be backwards-compatible.
   - [User Stories](https://docs.google.com/document/d/1wT8e8wBHMrSRHY4UF_60GCgyWGqvYye4THvaDARPySs/edit)
   - [Features](https://docs.google.com/document/d/1JNP6DgSK-c6KqveIhQk-n_HAw3hsZkL-okoleM43NgA/edit)
   - [Design Space](https://docs.google.com/document/d/1TpQlQYovCoX9FkpgsoxzdvZplghudHAiQOame30A-v8/edit)
-- [Frequently Asked Questions](FAQ.md)
+- [Frequently Asked Questions](docs/FAQ.md)
 
-## Usage
+## Setup
 
 Get the tool via
 
 ```sh
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+To start managing dependencies using dep, run the following from your project root directory:
 
 ```sh
 $ dep init
 $ dep ensure -update
 ```
 
-To update a dependency to a new version, you might run
+`dep init` will do the following:
+
+1. Look for [existing dependency management files](docs/FAQ.md#what-external-tools-are-supported) to convert
+1. Check if your dependencies use dep
+1. Identify your dependencies
+1. Back up your existing `vendor/` directory (if you have one) to
+`_vendor-TIMESTAMP/`
+1. Pick the highest compatible version for each dependency
+1. Generate [`Gopkg.toml`](Gopkg.toml.md) and `Gopkg.lock` files
+1. Install the dependencies in `vendor/`
+
+## Usage
+
+There is one main subcommand you will use: `dep ensure`. `ensure` synchronizes
+your dependencies in `vendor/` to make sure they match what's in your `import`s
```

Comment:
Close, but a little off the mark. Maybe:

`ensure` first makes sure `Gopkg.lock` is consistent with your imports and `Gopkg.toml`, and then that `vendor` has is poopulated with exactly what's described in `Gopkg.lock`.